apps/modernization-ui/src/apps/patient/data/address/AddressEntryFields.spec.tsx

```diff
@@ -133,4 +133,47 @@ describe('AddressEntryFields', () => {
             expect(queryByText('Use is required.')).not.toBeInTheDocument();
         });
     });
+
+    it('should invalidate Census Tract format', async () => {
+        const { getByLabelText, getByText, queryByText } = render(<Fixture />);
+
+        const censusTractInput = getByLabelText('Census tract');
+
+        const testCases = [
+            { value: '0000.00', valid: false },
+            { value: '0001.00', valid: false },
+            { value: '0001.01', valid: true },
+            { value: '1000.00', valid: false },
+            { value: '9999.99', valid: false },
+            { value: '9999.98', valid: true },
+            { value: '0001', valid: true },
+            { value: '9999', valid: true },
+            { value: '0000', valid: false },
+            { value: '9999.00', valid: false },
+            { value: '0001.99', valid: false },
+            { value: '1234.56', valid: true }
+        ];
```

Comment:
suggestion (non-blocking):  this kind of test can be simplified into a Parameterized test using [test.each](https://jestjs.io/docs/api#testeachtablename-fn-timeout)